src/plugins/mutes.js

```diff
@@ -18,5 +18,18 @@ export default function mutesPlugin () {
       mute,
       unmute
     })
+
+    function onModMute (ref) {
+      mp.emit('modMute', {
+        moderator: ref.m,
```

Comment:
I think this isn't right, I am seeing a `mi` property for the moderator ID in the browser:

![](http://i.imgur.com/0o5LBSj.png)